portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -374,6 +374,63 @@ definition {
 		AssertElementPresent(locator1 = "CPSAMLAdminIdentityProviderConnection#IDENTITY_PROVIDER_CONNECTION_ERROR_NO_MAPPING_SELECTED");
 	}
 
+	@description = "LPS-123218 TC-2 Assertion for: first name cannot be used for user matching even if LDAP import is enabled."
+	@priority = "4"
+	test AssertFirstNameCannotBeUsedForUserMatchingEvenWithLDAPEnabled {
+		property apacheds.enabled = "true";
+		property app.server.bundles.size = "0";
+		property test.name.skip.portal.instance = "SAML#AssertFirstNameCannotBeUsedForUserMatchingEvenWithLDAPEnabled";
+
+		PortalSettings.addLDAPServer(LDAPServerName = "ApacheDS Server");
+
+		PortalSettings.editLDAPServer(LDAPServerName = "ApacheDS Server");
+
+		PortalSettings.testLDAPConnection();
```

Comment:
SACRED: Could we combine these 3 LDAP macros into one?

Reply:
Good point! Created LDAP.macro for it.